API_compatible_README.md

```diff
@@ -0,0 +1,24 @@
+# API Compatible
+
+*Instructions are for setting up API for Audacious client on Ubuntu. The steps can be modified for other clients as well.*
+
+**SOURCE_1**: http://redmine.audacious-media-player.org/boards/1/topics/788
+
+1. Install dependencies from **SOURCE_1** then clone the repo and install audacious.
+2. Before installing audacious-plugins, edit the file `audacious-plugins/src/scrobbler2/scrobbler.h` to update the following settings on lines L26-L28.  
+  - `SCROBBLER_API_KEY` to the key given on "listenbrainz.org/user/import" page.
+  - `SCROBBLER_URL` to "http://last.fm/2.0/".  
+  Now compile and install the plugins from the instructions given in **SOURCE_1**.
+3. Register an application on musicbrainz with the following callback URL "http://www.last.fm/login/musicbrainz/post"
```

Comment:
Why is this step required? Is it because the URL that audacious generates starts as `http://last.fm/api/auth...`? I don't know if it's a good idea for us to respond to requests like this with the listenbrainz site. Perhaps instead our instructions could say something like "When you get a URL from your application which starts `http://last.fm/api/auth`, replace it with `http://listenbrainz.org/.../../...`"